iconMappings.ts

```diff
@@ -0,0 +1,107 @@
+export default {
+	accountCircleOutline: { file: 'account-circle-outline.svg', fa_icon: 'far-user-circle' },
```

Comment:
```suggestion
	accountCircleOutline: { file: 'account-circle-outline.svg', fa_icon: 'far-circle-user' },
```
imo this is the propper FA icon name.

Reply:
It depends
The `far-user` is the better representation of the mdi one. And we use both variants in the design mockups. I will double-check which one to use. The Header and Simplesettings navigation use the one without circle